bip-0039/indonesian.txt

```diff
@@ -0,0 +1,2048 @@
+pedesaan
+pedoman
+peduli
+pegawai
+pegunungan
+pejabat
+pejuang
+pekan
+pekerjaan
+pelayanan
+pelepasan
+pelindung
+pelopor
+peluncuran
+pemain
+pembangunan
+pemerintah
+pemimpin
+pemrograman
+pemuda
+penampilan
+pencarian
+penduduk
+penerbangan
+pengguna
+peningkatan
+penjualan
+penonton
+pensiun
+penting
+penulis
+penyanyi
+peperangan
+pepohonan
+perang
+perbedaan
+percaya
+perdagangan
+perempuan
+perfilman
+pergi
+perhatian
+periode
+perjalanan
+perkembangan
+perlu
+permukaan
+pernah
+perolehan
+perpustakaan
+persen
+pertama
+perusahaan
+perwira
+pesawat
+peserta
+pesisir
+pesta
+petani
+peternakan
+peti
+petunjuk
+pewaris
+piala
+pidato
+pihak
+pikiran
+pilihan
+pimpinan
+pindah
+pinggiran
+pinjaman
+pintu
+piringan
+pisau
+pita
+plastik
+plat
+plot
+plus
+pohon
+poin
+pokok
+pola
+politik
+pondok
+ponsel
+populer
+poros
+posisi
+potensi
+potongan
+potret
+prajurit
+praktik
+pranala
+prasasti
+predator
+prefektur
+presiden
+pria
+pribadi
+primordial
+prinsip
+prioritas
+produksi
+profesional
+progresif
+promosi
+properti
+proses
+protein
+provinsi
+proyek
+psikologi
+puas
+publik
+pucat
+puing
+puisi
+pujian
+pukul
+pula
+pulih
+puluh
+punah
+puncak
+punggung
+punk
+punya
+pupuk
+pura
+purba
+pusat
+pustaka
+putaran
+putera
+putih
+putra
+putus
+racun
+radius
+ragam
+ragu
+rahasia
+raih
+raja
+rajin
+raksasa
+rakyat
+ramai
+rambut
+ranah
+rancangan
+rangka
+rantai
+rapat
+rapi
+rasa
+rasio
+rasul
+rating
+ratusan
+rawa
+raya
+reaksi
+realitas
+redaksi
+referensi
+reformasi
+regional
+reguler
+rekaman
+rekor
+rekreasi
+rektor
+relasi
+relevan
+religius
+remaja
+rempah
+renang
+rencana
+rendah
+renovasi
+rentang
+reproduksi
+republik
+reruntuhan
+resep
+resmi
+resolusi
+respon
+restoran
+revisi
+revolusi
+rezim
+ribu
+rilis
+rincian
+ringan
+rintisan
+riset
+risiko
+riwayat
+roda
+rohani
+roket
+rokok
+romantis
+rombongan
+ronde
+rongga
+rotasi
+roti
+ruang
+ruas
+rudal
+rugi
+rujukan
+rumah
+rumit
+rumput
+rumus
+runtuh
+rupa
+rupiah
+rusak
+rute
+rutin
+saat
+sabda
+sabuk
+sadar
+saham
+sains
+saja
+sakit
+saksi
+sakti
+salah
+saleh
+saling
+salju
+saluran
+sama
+sambil
+sampai
+samurai
+sana
+sandiwara
+sangat
+sanksi
+santri
+sapi
+sarana
+sari
+sarjana
+sasaran
+sastra
+satelit
+satu
+satwa
+saudara
+saus
+sawah
+sawit
+saya
+sayuran
+seakan
+sebagai
+sebesar
+sebuah
+secara
+sedang
+sederhana
+sedikit
+seekor
+segala
+segera
+segi
+segmen
+sehari
+sehingga
+sehubungan
+seimbang
+seiring
+sejak
+sejenis
+sejumlah
+sekarang
+sekelompok
+sekitar
+sekolah
+sekretaris
+seksual
+sektor
+sekutu
+selama
+selesai
+selisih
+seluruh
+semakin
+sembilan
+sementara
+seminggu
+semoga
+sempat
+semua
+senang
+sendiri
+sengaja
+seni
+senjata
+sensus
+sentral
+senyawa
+seolah
+seorang
+sepanjang
+seperti
+sepuluh
+serangan
+serba
+serentak
+sering
+serta
+serupa
+server
+sesama
+seseorang
+sesi
+sesuatu
+setahun
+setelah
+setiap
+setuju
+seumur
+sewaktu
+siang
+siapa
+siaran
+sibuk
+sidang
+sifat
+signifikan
+sihir
+sikap
+siklus
+silat
+silinder
```

Comment:
Technical term for most native speakers.

Reply:
I can agree to this.